### PR TITLE
Calo DB next round of changes

### PR DESCRIPTION
**Pull request by sophiemiddleton**
The next round of changes for the calorimeter DB information.

### CODE REVIEW

**Comment by kutschke** [comment]
@sophiemiddleton  is this PR still infrastructure or will it modify the behaviour of nightly validation?

**Comment by sophiemiddleton** [comment]
its still infrastructure

**Comment by sophiemiddleton** [comment]
@FNALbuild run build test

File: DbTables/inc/CalCosmicT0Align.hh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    private:
+      CaloSiPMId   _roid;
+      float _Tcorr;
+      float _Terr;
+      float _chisq;
+    };
+
+    constexpr static const char* cxname = "CalCosmicT0Align";
+
+    CalCosmicT0Align():DbTable(cxname,"cal.cosmicT0align","roid,Tcorr,Terr,chisq"){}
```
Comment (rlcee) [inline DbTables/inc/CalCosmicT0Align.hh:34]: In the quotes here is the database table name and column names, which are case insensitive, please make these "T" into "t".  We will want to make this change in existing tables, but that can wait for another PR.

File: DbTables/inc/CalCosmicT0Align.hh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    const Row& row(std::uint16_t roid) const { return _rows.at(roid); }
```
Comment (rlcee) [inline DbTables/inc/CalCosmicT0Align.hh:36]: In the other tables, we had a CaloSIPMId here

File: CaloConditions/src/CalCalibMaker.cc [hunk 3]
```diff
@@ -1,29 +1,71 @@
...
   typedef std::shared_ptr<CalCalib> ptr_t;
 
+/*******************************************************************/
   ptr_t CalCalibMaker::fromFcl() {
-  CalCalibPar nominal(_config.ADC2MeV(), _config.timeoffset(), _config.algID());
+  
+  if (_config.verbose()) {
+    std::cout << "CRalCalibMaker::fromFcl making nominal CalCalib\n";
+  }
+  CalCalibPar nominal(_config.ADC2MeV(), _config.timeoffset(), _config.ECombAlgID());
```
Comment (rlcee) [inline CaloConditions/src/CalCalibMaker.cc:16]: Is the order of arguments incorrect here?

File: CaloConditions/src/CalCalibMaker.cc [hunk 4]
```diff
@@ -1,29 +1,71 @@
...
 
   size_t nChan = CaloConst::_nChannel;
 
+  if (_config.verbose()) {
+    std::cout << "CalCalibMaker::fromFcl filling " << nChan << " channels\n";
+    std::cout << "CalCalibMaker::fromFcl nominal " << fixed << setprecision(3)
+         << setw(10) << nominal.ADC2MeV() << setprecision(3) << setw(10)
+         << nominal.timeOffset() << setprecision(3) << setw(10)
```
Comment (rlcee) [inline CaloConditions/src/CalCalibMaker.cc:24]: order here should probably follow argument order

**Review by rlcee — CHANGES_REQUESTED** [review]
One general comment.  The idea of embracing CaloSiPMId is that these: "std::uint16_t roid" become "CaloSiPMId roid".   It is up to you and the other calo coders how to do this, but that is the idea.  A fe w other comments inline.

**Comment by kutschke** [comment]
@FNALbuild run build test

File: DbTables/inc/CalCosmicT0Align.hh (hunk 2 above)
Comment (sophiemiddleton) [reply DbTables/inc/CalCosmicT0Align.hh:36]: I dont see that in other tables, but maybe it needs to be implemented for all?

Comment (rlcee) [reply DbTables/inc/CalCosmicT0Align.hh:36]: I was comparing to CalEnergyCalib, which uses it, but it looks like that is the only table which does.  But I think this is part of the general comment, how much do you want to embrace CaloSiPMID, and how much right now?  I won't insist in this PR, but eventually there should be a consistent pattern.   I also see some tables use int and some uint16_t

**Comment by sophiemiddleton** [comment]
@FNALbuild run build test

File: DbTables/inc/CalCosmicEnergyCalib.hh [hunk 5]
```diff
@@ -37,7 +37,8 @@ namespace mu2e {
 
     CalCosmicEnergyCalib():DbTable(cxname,"cal.cosmicenergycalib","roid,EPeak,ErrEPeak,Width,ErrWidth,chisq"){}
 
-    const Row& row(std::uint16_t roid) const { return _rows.at(roid); }
+    const Row& row(CaloSiPMId  _roid) const {
```
Comment (rlcee) [inline DbTables/inc/CalCosmicEnergyCalib.hh:40]: Sorry one more - this underscore is misleading

Comment (sophiemiddleton) [reply DbTables/inc/CalCosmicEnergyCalib.hh]: fixed

**Comment by sophiemiddleton** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
@sophiemiddleton The CI found whitespace issues. Please address these.  Thanks.

**Review by rlcee — APPROVED** [review]

**Comment by kutschke** [comment]
@sophiemiddleton Ray has approved this now.  I will wait for the whitespace fixes.

**Comment by sophiemiddleton** [comment]
@FNALbuild run build test

**Review by kutschke — APPROVED** [review]
Thanks for the final fixes.  I will merge now.

**Pull request merged**